src/server/server_family.cc

```diff
@@ -1262,6 +1264,9 @@ void ServerFamily::Cluster(CmdArgList args, ConnectionContext* cntx) {
     std::string msg = BuildClusterNodeReply(cntx);
     (*cntx)->SendBulkString(msg);
     return;
+  } else if (sub_cmd == "READONLY") {
+    (*cntx)->SendOk();
```

Comment:
you have not checked with https://github.com/mediocregopher/radix that it works are not you? :)

it does not fix the issue because this is not a subcommand. it's a separate command that exists only in cluster mode. so you need to register it conditionally only if we run in emulated mode.

Reply:
correct, have not tested with radix.